core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/DummyTextAnnotationGenerator.java

```diff
@@ -195,113 +212,151 @@ public static TextAnnotation generateBasicTextAnnotation(int numSentences) {
 
     private static String[] allPossibleViews = new String[] {ViewNames.POS, ViewNames.LEMMA,
             ViewNames.SHALLOW_PARSE, ViewNames.PARSE_GOLD, ViewNames.SRL_VERB, ViewNames.NER_CONLL,
-            ViewNames.PSEUDO_PARSE_STANFORD };
+            ViewNames.PSEUDO_PARSE_STANFORD};
 
-    public static TextAnnotation generateAnnotatedTextAnnotation(boolean withNoisyLabels) {
-        return generateAnnotatedTextAnnotation(allPossibleViews, withNoisyLabels);
+    public static TextAnnotation generateAnnotatedTextAnnotation(boolean withNoisyLabels,
+                                                                 int sentenceLength) {
+        return generateAnnotatedTextAnnotation(allPossibleViews, withNoisyLabels, sentenceLength);
     }
 
     public static TextAnnotation generateAnnotatedTextAnnotation(String[] viewsToAdd,
-            boolean withNoisyLabels) {
-        TextAnnotation ta = BasicTextAnnotationBuilder.createTextAnnotationFromTokens(annotatedTokenizedStringArray);
+                                                                 boolean withNoisyLabels, int sentenceLength) {
```

Comment:
I don't think `sentenceLength` is the appropriate variable name. Maybe `sentenceNum`?